utils/namespaces.js

```diff
@@ -0,0 +1,33 @@
+import { sortBy } from '@/utils/sort';
+
+// This method is used by the NameNsDescription component and by the create/edit
```

Comment:
Rather than focus on the semantic usage of the function, it may be more appropriate to define the scope, e.g.
```js
/*
 * Get only (add quality of the entries here) namespaces
 */
```

PS: There's a nice autocompletition while typing `/**`, plus this is more the jsdoc format for code docs headers.